CHANGELOG.md

```
@@ -12,6 +12,34 @@ This changelog summarizes releases for the following packages:
+
+## @effectify/react-query
+
+## 1.0.1-beta.0 (2026-08-31)
+
+## @effectify/react-router
+
+## 0.6.1-beta.0 (2026-08-31)
+
+## @effectify/react-router-better-auth
+
+## 0.5.13-beta.0 (2026-08-31)
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)